src/common/evaluation/QA/eval_depthmap_models/src/evaluate.py

```diff
@@ -132,8 +132,17 @@ def get_prediction(MODEL_PATH, dataset_evaluation):
 
     print("Using {} artifact files for evaluation.".format(len(paths_evaluation)))
 
+    standing = load_model(FILTER_CONFIG.NAME)
+    new_paths_evaluation = paths_evaluation
+
+    if FILTER_CONFIG.SWITCH == True:
```

Comment:
Notice that the parameter `SWITCH` does not exist anymore

```suggestion
    if FILTER_CONFIG.IS_ENABLED:
```